docs/en/operations/access_rights.md

```diff
@@ -104,5 +104,10 @@ The user can get a list of all databases and tables in them by using `SHOW` quer
 
 Database access is not related to the [readonly](settings/permissions_for_queries.md#settings_readonly) setting. You can't grant full access to one database and `readonly` access to another one.
 
+## ZooKeeper Authenticaton {#zookeeper-authentification}
+
+When you use [replication](table_engines/replication.md), ClickHouse stores replica's meta information in a ZooKeeper infrastructure. For more safety, set up authentification for ZooKeeper.
+
+ClickHouse supports the `digest` [ACL scheme](https://zookeeper.apache.org/doc/current/zookeeperProgrammers.html#sc_ZooKeeperAccessControl) of the ZooKeeper security subsystem. The [zookeeper](server_settings/settings.md#server-settings_zookeeper) server configuration section allows setting up the username and password required for authenticating on the ZooKeeper server.
```

Comment:
needs some example

Reply:
There is an example in the zookeeper server setting description.